cli/package.json

```diff
@@ -38,7 +38,7 @@
     "dayjs": "^1.10.4",
     "debug": "^4.3.2",
     "enquirer": "^2.3.6",
-    "eventemitter2": "^6.4.3",
+    "eventemitter2": "6.4.7",
```

Comment:
This is causing CI to fail, 6.4.8 has a breaking change for types. I locked to 6.4.7 for now. Because we did `^6.4.3` before (notice `^`) we got the highest `6.4.x`. 

They've got an open issue: https://github.com/EventEmitter2/EventEmitter2/issues/292